src/tests/conftest.py

```diff
@@ -435,3 +441,111 @@ def _fetcher(scopes):
         return token.serialize()
 
     return _fetcher
+
+
+# ---| >> START no display check>> FIETSPAALTJES  >>no display check >> |---#
+
+
+@pytest.fixture()
+def fietspaaltjes_schema(fietspaaltjes_schema_json) -> DatasetSchema:
+    return DatasetSchema.from_dict(fietspaaltjes_schema_json)
+
+
+@pytest.fixture()
+def fietspaaltjes_container_model(filled_router):
+    # Using filled_router so all urls can be generated too.
+    return filled_router.all_models["fietspaaltjes"]["fietspaaltjes"]
+
+
+@pytest.fixture()
+def fietspaaltjes_schema_json() -> dict:
+    """Fixture to return the schema json for """
+    path = HERE / "files/fietspaaltjes.json"
+    return json.loads(path.read_text())
+
+
+@pytest.fixture()
+def fietspaaltjes_dataset(fietspaaltjes_schema_json) -> Dataset:
+    return Dataset.objects.create(
+        name="fietspaaltjes", schema_data=fietspaaltjes_schema_json
+    )
+
+
+@pytest.fixture()
+def fietspaaltjes_data(fietspaaltjes_container_model):
+    return fietspaaltjes_container_model.objects.create(
+        id="Fietsplaatje record met display",
+        geometry="POINT (123207.6558130105 486624.6399002579)",
+        street="Weesperplein",
+        at="Geschutswerf",
+        area="Amsterdam-Centrum",
+        score_2013=None,
+        score_current="reference for DISPLAY FIELD",
+        count=6,
+        paaltjes_weg=["nu paaltje(s)"],
+        soort_paaltje=["paaltje(s) ong. 75cm hoog", "verwijderde paaltjes"],
+        uiterlijk=["rood/wit"],
+        type=["vast", "uitneembaar"],
+        ruimte=["Voldoende: 1.6m of meer"],
+        markering=["markering ontbreekt", "onvoldoende markering"],
+        beschadigingen=None,
+        veiligheid=["overzichtelijke locatie"],
+        zicht_in_donker=["onvoldoende reflectie op paal"],
+        soort_weg=["rijbaan fiets+auto", "fietspad"],
+        noodzaak=["nodig tegen sluipverkeer"],
+    )
+
+
+@pytest.fixture()
+def fietspaaltjes_schema_no_display(
+    fietspaaltjes_schema_json_no_display,
+) -> DatasetSchema:
+    return DatasetSchema.from_dict(fietspaaltjes_schema_json_no_display)
+
+
+@pytest.fixture()
+def fietspaaltjes_container_model_no_display(filled_router):
```

Comment:
Dit is toch geen "container_model" ? (copy-paaste foutje in naam)

Reply:
Ja klopt, geen container, een copy-paste overblijfsel, ik pas de naam aan